builtin/providers/aws/resource_aws_waf_ipset_test.go

```diff
@@ -0,0 +1,104 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/waf"
+)
+
+func TestAccAWSWafIPSet_basic(t *testing.T) {
```

Comment:
I'd like to see another test that tests for the `WAFNonexistentItemException` causing a non-empty plan. If you search in the AWS folder for `_disappears`, you will see how this looks